server/google_apis/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: cloud_run_google_apis
+publish_to: none # let's not accidentally publish this to pub.dev
```

Comment:
I think I'd s/let's not accidentally publish this to pub.dev/not intended for publication to pub.dev/

Reply:
ack